### PR TITLE
Feature/yelb app

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:* Adding Ingress to yelb and ho11y apps in `team-geordi`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: teams/team-geordie/prod/ho11y/values.yaml [hunk 1]
```diff
@@ -2,5 +2,7 @@ spec:
   destination:
     server: https://kubernetes.default.svc
   source:
-    repoURL: https://github.com/aws-samples/eks-blueprints-workloads
-    targetRevision: main
\ No newline at end of file
+    repoURL: https://github.com/elamaran11/eks-blueprints-workloads
```
Comment (shapirov103) [inline teams/team-geordie/prod/ho11y/values.yaml]: This should be reverted. Applies elsewhere.

File: teams/team-geordie/prod/yelb/values.yaml [hunk 2]
```diff
@@ -2,5 +2,7 @@ spec:
   destination:
     server: https://kubernetes.default.svc
   source:
-    repoURL: https://github.com/aws-samples/eks-blueprints-workloads
-    targetRevision: main
\ No newline at end of file
+    repoURL: https://github.com/elamaran11/eks-blueprints-workloads
+    targetRevision: feature/Yelb-app
+  ingress:
+    host: teamblueprints.com
```
Comment (shapirov103) [inline teams/team-geordie/prod/yelb/values.yaml]: why hardcoded?

**Review by shapirov103 — COMMENTED** [review]
Thank you @elamaran11  for addressing the ingress side. There are a couple of entries that you most likely missed to clean up before pushing.

File: teams/team-geordie/prod/ho11y/values.yaml (hunk 1 above)
Comment (elamaran11) [reply teams/team-geordie/prod/ho11y/values.yaml]: Fixed this.

File: teams/team-geordie/prod/yelb/values.yaml (hunk 2 above)
Comment (elamaran11) [reply teams/team-geordie/prod/yelb/values.yaml]: Good Catch. Fixed and remove it.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**